CMakeLists.txt

```diff
@@ -118,12 +118,11 @@ pkg_check_modules(UUID IMPORTED_TARGET REQUIRED uuid>=2.23)
 pkg_check_modules(CGRAPH IMPORTED_TARGET libcgraph>=2.30)
 pkg_check_modules(GVC IMPORTED_TARGET libgvc>=2.30)
 pkg_check_modules(LIBUSB IMPORTED_TARGET libusb-1.0>=1.0.23)
-pkg_check_modules(XIL IMPORTED_TARGET libxil)
 pkg_check_modules(NANOMSG IMPORTED_TARGET nanomsg)
 if(NOT NANOMSG_FOUND)
     pkg_check_modules(NANOMSG IMPORTED_TARGET libnanomsg>=1.0.0)
 endif()
-pkg_check_modules(RE IMPORTED_TARGET re>=0.5.6)
+pkg_check_modules(RE IMPORTED_TARGET re>=2.9.0)
```

Comment:
why is this necessary?

Reply:
We switched to a newer libre version (for the `rtp` node-type) as Ubuntu 22.04 ships with this new version and it broke our build. I think you fixed the RTP stuff on ernie some while ago..